live/global.ini

```
@@ -288,10 +288,10 @@ BoardingGate_Verifying=Überprüfung
 Boarding_Gate_Access_Granted=Zugang gewährt
 Boarding_Gate_Automatic_door=Automatik-Tor
 Boarding_Gate_Caution=Vorsicht
-Boarding_Gate_Check_ID=Bitte deine ID überprüfen
+Boarding_Gate_Check_ID=Bitte Ihre ID überprüfen
 Boarding_Gate_Confidential=Vertraulich
-Boarding_Gate_Press_ID=Drücke ID auf dem Bildschirm
-Boarding_Gate_Proceed=Das Gate passieren
+Boarding_Gate_Press_ID=Drücken Sie ID auf dem Bildschirm
```

```suggestion
Boarding_Gate_Press_ID=Drücken Sie die ID auf dem Bildschirm
```

---

Ne, das passt nicht ^^

---

Doch xD 

Drücken Sie Ausweis auf Bildschirm. 

Sehr tolles Deutsch :-D

---

Alternativ würde auch 
```suggestion
Boarding_Gate_Press_ID=Drücken Sie ihre ID auf dem Bildschirm
```